code/parse_test.go

```diff
@@ -410,7 +410,7 @@ func TestParseKinesisFirehoseLogs(t *testing.T) {
 	logs := parseCloudWatchLogs(cloudWatchEvent)
 
 	localTime := time.Local
-	timeValue := time.Date(2021, time.April, 26, 11, 15, 15, 22800000, time.Local)
+	timeValue := time.Date(2021, time.April, 26, 11, 15, 15, 228000000, time.Local)
```

Comment:
What is this change? 
I remember we had discussion that tests where failing if we create time from string. You can also try to convert it to UTC and try.